test/integration/simulacron.js

```diff
@@ -0,0 +1,290 @@
+    method: 'DELETE'
+  };
+  makeRequest(options, function(data) {
+    callback();
+  }).end();
+}
+
+SimulacronCluster.prototype.clearLog = function(callback) {
+  var self = this;
+  var destroyClusterPath = '/log/%d';
+  var options = {
+    host: self.baseAddress,
+    path: encodeURI(util.format(destroyClusterPath, self.id)),
+    port: self.port,
+    method: 'DELETE'
+  };
+  makeRequest(options, function(data) {
+    callback(null);
+  }).end();
+}
+
+SimulacronCluster.prototype.primeQueryWithEmptyResult = function(queryStr, callback) {
+  var self = this;
+  var primeQueryPath = '/prime/%d';
+  var options = {
+    host: self.baseAddress,
+    path: encodeURI(util.format(primeQueryPath, self.id)),
+    port: self.port,
+    method: 'POST',
+    headers: { "Content-Type": "application/json" }
+  };
+  var body = {
+    when: {
+      query: queryStr
+    },
+    then: {
+      result: "success",
+      delay_in_ms: 0,
+      rows: [],
+      column_types: {}
+    }
+  };
+  var request = makeRequest(options, function(data) {
+    callback();
+  });
+  request.write(JSON.stringify(body));
+  request.end();
+}
+
+SimulacronCluster.prototype.findNode = function(nodeAddress) {
+  var self = this;
+  var node = null;
+  self.dcs.find(function(dataCenter) {
+    var nodeItem = dataCenter.nodes.find(function(dcNode) {
+      return dcNode.address === nodeAddress;
+    });
+    if (nodeItem === undefined) {
+      return false;
+    }else {
+      node = {
+        nodeId: nodeItem.id,
+        dataCenterId: dataCenter.id
+      };
+      return true;
+    }
+  });
+  return node;
+}
+
+SimulacronCluster.prototype.getContactPoints = function(dataCenterId) {
+  var dcId = dataCenterId = typeof dataCenterId === 'undefined' ? 0 : dataCenterId;
+  return this.dcs[dcId].nodes.map(function (node) {
+    return node.address;
+  });
+}
+
+SimulacronCluster.prototype.queryNodeLog = function(nodeAddress, callback) {
+  var self = this;
+  var node = self.findNode(nodeAddress);
+  if (node === null) {
+    throw new Error('invalid node address: ' + nodeAddress);
+  }
+
+  var queryLog = '/log/%d/%d/%d';
+  var options = {
+    host: self.baseAddress,
+    path: encodeURI(util.format(queryLog, self.id, node.dataCenterId, node.nodeId)),
+    port: self.port,
+    method: 'GET'
+  };
+  makeRequest(options, function(data) {
+    callback(data.data_centers[0].nodes[0].queries);
+  }).end();
+}
+
+SimulacronCluster.prototype.stopNode = function(nodeAddress, callback) {
+  var self = this;
+  var node = self.findNode(nodeAddress);
+  if (node === null) {
+    callback(new Error('invalid node address: ' + nodeAddress));
+  }
+
+  var stopNodePath = '/listener/%d/%d/%d?type=stop';
+  var options = {
+    host: self.baseAddress,
+    path: encodeURI(util.format(stopNodePath, self.id, node.dataCenterId, node.nodeId)),
+    port: self.port,
+    method: 'DELETE'
+  };
+  makeRequest(options, function(data) {
+    callback(data);
+  }).end();
+}
```

Comment:
There are other linting issues: 
- missing semicolon
- no-else-return
- Use of console

You can run eslint locally: `npm run eslint`